src/norm/objutils.nim

```diff
@@ -77,15 +77,19 @@ proc toSignatureRepr(def: NimNode): SignatureRepr =
       result.name = $def[0][1]
       result.exported = true
     of nnkPragmaExpr:
-      expectKind(def[0][0], {nnkIdent, nnkPostfix})
+      expectKind(def[0][0], {nnkIdent, nnkSym, nnkPostfix})
       case def[0][0].kind
       of nnkIdent:
         result.name = $def[0][0]
+      of nnkSym:

```

Comment:
Why not just modify the condition above to `of nnkIdent, nnkSym`?

Also, could you please explain what the difference between symbol and identifier is?

Reply:
Making the change to the case statement.

When using use the `db` macro, the name of the field is given in the source code, a `nnkIndent`. When using dbAddObject or dbAddCollection, the object already exist, so the field is a referenced symbol, a `nnkSym`. With this change, both the direct definition and the indirect reference works in `toObjRepr`.